catalyst/core/callbacks/early_stop.py

```diff
@@ -0,0 +1,59 @@
+from typing import TYPE_CHECKING  # isort:skip
+
+from catalyst.core import Callback, CallbackOrder
+
+if TYPE_CHECKING:
+    from catalyst.core import _State
+
+
+class CheckRunCallback(Callback):
```

Comment:
Looks like now we will have two add (and remove after) extra lines in config to check if it works.
Maybe you can add this callback as default if `args.check` present on experiment constcurtion phase?

Reply:
already